packages/zcli-themes/tests/functional/preview.test.ts

```diff
@@ -87,12 +91,9 @@ describe('themes:preview', function () {
 
         try {
           await PreviewCommand.run([baseThemePath])
-        } catch {
           expect(ctx.stdout).to.contain(`Validation error ${baseThemePath}/templates/home_page.hbs:10:6`)
           expect(ctx.stdout).to.contain("'searcsh' does not exist")
```

Comment:
nit: I know that this was added in a previous PR, but this should be altered from `searcsh` -> `search`